lib/hammer_cli/apipie/options.rb

```diff
@@ -34,5 +34,18 @@ def get_option_value(opt_name)
       end
     end
 
+    def add_default_options(options, explicit_options)
+      defaults = {}
+      options.each do |option|
+        option = option.name.to_s
+        value = HammerCLI::Defaults.get_defaults(option) unless explicit_options["option_"+option]
+        if value
+          defaults["option_" + option] = value
+          logger.info(_("You are using the following default parameter:  #{option} = #{value}"))
```

Comment:
Either use % and put arguments outside the gettext function or leave the string untranslated. I'm fine with the latter. We usually don't translate log messages.